test/e2e/cron/cron_test.go

```diff
@@ -0,0 +1,147 @@
+package cron
+
+import (
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	"github.com/argoproj/argo/test/e2e/fixtures"
+	"github.com/argoproj/argo/workflow/common"
+	"github.com/argoproj/pkg/humanize"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"testing"
+	"time"
+)
+
+type CronSuite struct {
+	fixtures.E2ESuite
+}
+
+func (s *CronSuite) TestBasic() {
+	s.T().Parallel()
+	s.Given().
+		CronWorkflow("@testdata/basic.yaml").
+		When().
+		CreateCronWorkflow().
+		Wait(1 * time.Minute).
```

Comment:
anyway to reduce this time to <30s?

Reply:
Please see https://github.com/argoproj/argo/pull/1758#discussion_r363346602